<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetectSearchItemDialog</class>
 <widget class="QDialog" name="SetectSearchItemDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select/View item</string>
  </property>
  <widget class="QPushButton" name="ButtonSelectAndView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>500</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Select and view</string>
   </property>
   <property name="icon">
    <iconset resource="DisplayImage.qrc">
     <normaloff>:/Qtres/Select.png</normaloff>:/Qtres/Select.png</iconset>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetItems">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>721</width>
     <height>381</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>Page</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Layer</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>LibID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Library</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>581</width>
     <height>101</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QLineEdit" name="lineEditTopName">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>By name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>By ID</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxTopID">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999999999</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSortByID">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort and top</string>
    </property>
    <property name="icon">
     <iconset resource="DisplayImage.qrc">
      <normaloff>:/Qtres/SortNumber.png</normaloff>:/Qtres/SortNumber.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSortByName">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>60</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort and top</string>
    </property>
    <property name="icon">
     <iconset resource="DisplayImage.qrc">
      <normaloff>:/Qtres/SortNumber.png</normaloff>:/Qtres/SortNumber.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSortByLibID">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort by LibID</string>
    </property>
    <property name="icon">
     <iconset resource="DisplayImage.qrc">
      <normaloff>:/Qtres/SortText.png</normaloff>:/Qtres/SortText.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="ButtonOnlyView">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>500</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Only view</string>
   </property>
   <property name="icon">
    <iconset resource="DisplayImage.qrc">
     <normaloff>:/Qtres/Display.png</normaloff>:/Qtres/Display.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>500</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="icon">
    <iconset resource="DisplayImage.qrc">
     <normaloff>:/Qtres/Cancel.png</normaloff>:/Qtres/Cancel.png</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="DisplayImage.qrc"/>
 </resources>
 <connections/>
</ui>
